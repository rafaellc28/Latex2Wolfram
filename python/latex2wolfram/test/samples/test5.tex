% Test integral expression with both lower and upper bound
(\int_{0}^{pi}\sin(x)dx)
