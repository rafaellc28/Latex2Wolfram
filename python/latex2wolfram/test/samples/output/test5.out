(integrate sin(x) dx from x=0 to pi)
